src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/CqlProxyRetryPolicy.java

```diff
@@ -20,10 +21,13 @@
  * the intermediate layer is unavailable for some reason. In all other cases, this will rethrow the
  * exception to avoid retrying the requests at the driver level.
  */
+@RegisterForReflection
 public class CqlProxyRetryPolicy implements RetryPolicy {
   private static final Logger LOG = LoggerFactory.getLogger(CqlProxyRetryPolicy.class);
   private final String logPrefix;
 
+  private static final int MAX_RETRIES = 3;
```

Comment:
should this be configurable?

Reply:
I think we need to be able to tune this. Assuming that one of the possible errors is a timeout, the timeout value is going to be settable in profile. We need to be able to tune the two properties together so we can have control over the max time we might spend (timeout X retries?).

Reply:
There is no driver configuration to set maximum retries. Set to 3 based on Aaron input. One option is to use System.getProperties().

Reply:
right, we'd have to add our own property